DESCRIPTION

```diff
@@ -1,8 +1,8 @@
 Package: RcppMsgPack
 Type: Package
-Title: 'MsgPack' C++ Header Files and R interface functions
-Version: 0.2.0
-Date: 2017-09-07
+Title: 'MsgPack' C++ Header Files and Interface Functions for R
+Version: 0.2.1
```

Comment:
Build up to a new release, I usually add a layer so that we can do 0.2.0.1, then 0.2.0.2, ... to eventually converge on 0.2.1.  

No need to change now though.